airesources/Go/src/hlt/entity.go

```diff
@@ -182,7 +182,13 @@ func IntToDockingStatus(i int) DockingStatus {
 
 // Thrust generates a string describing the ship's intension to move during the current turn
 func (ship Ship) Thrust(magnitude float64, angle float64) string {
-	return fmt.Sprintf("t %s %s %s", strconv.Itoa(ship.ID), strconv.Itoa(int(magnitude)), strconv.Itoa(int(angle)))
+	boundedAngle := int(angle)
+	if boundedAngle < 0 {
```

Comment:
Generally you want to avoid branches (if statements). I don't know if % in golang can result in negative numbers or not - various languages implement it differently. Just in case it can be negative, do boundedAngle = ((boundedAngle % 360) + 360) % 360. Otherwise you can just go with boundedAngle % 360 directly.

Reply:
(% is also more robust than current way that handles only limited input range)

Reply:
% in go can't handle negatives